pylib/cqlshlib/test/test_cqlsh_completion.py

```diff
@@ -1154,7 +1154,7 @@ def test_complete_in_alter_table(self):
                             other_choices_ok=True)
         self.trycompletions('ALTER TABLE new_table ADD col int C', immediate='HECK ')
         self.trycompletions('ALTER TABLE new_table ADD col int CHECK ',
-                            choices=['<identifier>', '<quotedName>', 'JSON', 'LENGTH', 'NOT_NULL', 'OCTET_LENGTH', 'REGEXP'],
```

Comment:
we can not do "NOT NULL" here ... it just does not play together with space in it. When "NOT" is suggested and a user types it and pushes "<tab>", "NULL" will be added automatically. At least something ...